examples/training/conllu.py

```diff
@@ -290,9 +289,9 @@ def get_token_conllu(token, i):
     return "\n".join(lines)
 
 
-Token.set_extension("get_conllu_lines", method=get_token_conllu)
-Token.set_extension("begins_fused", default=False)
-Token.set_extension("inside_fused", default=False)
+Token.set_extension("get_conllu_lines", method=get_token_conllu, force=True)
```

Comment:
Can we just move these inside the `main` function? Or does this not solve the problem? We typically avoid `force=True`, since this can lead to unexpected results and should really just be a last resort.

Reply:
Sure. The other script already had `force` set to `True`, but I can change both!